### ISSUE TITLE
Impossible to reach Outlook add-ins with keyboard navigation in Outlook for Mac app (new UI version)

### ISSUE THREAD

**Issue by ctruong9** [OfficeDev/office-js#2396]
## Expected Behavior
After keyboard navigating through the Outlook for Mac (new UI version) app, and pressing the link/button to load an add-in, the keyboard focus should automatically go into the add-in.

## Current Behavior
After keyboard navigating through the Outlook for Mac (new UI version) app, and pressing the link/button to load an add-in, the keyboard focus stays on the same link/button. Moreover, trying to navigate into the add-in from there is impossible. The result of  keyboard navigating through the app (using tab press) is a loop around the Outlook application without ever reaching any add-in that is loaded.

## Steps to Reproduce, or Live Example

* Link to live example: ______

* Additional details:
Steps to repro:
    1. Open Outlook for Mac (new UI version) app
    2. Ensure you have any Outlook add-in to load
    2. Click on an email in the inbox
    3. Using your keyboard to navigate with tab or shift+tab, go to an Outlook add-in
    4. Press space or enter to activate and load the add-in
    5. Notice the keyboard focus stays on the add-in link/button
 
## Context

This is causing Accessibility issues in which users who rely on keyboard navigation is unable to reach Outlook add-ins.

## Your Environment

* Platform [PC desktop, Mac, iOS, Office Online]: Mac
* Host [Excel, Word, PowerPoint, etc.]: Outlook for Mac (new UI version)
* Office version number: 16.52
* Operating System: macOS Catalina
* Browser (if using Office Online): N/A

## Useful logs

- [ ] Console errors
- [ ] Screenshots
- [ ] Test file (if only happens on a particular file)

**Comment by exextoc** [comment]
@ctruong9 Can you please provide us a video to better understand the UI experience you are seeing with tab navigation? Thanks!

**Comment by ctruong9** [comment]
It this screen recording, you can see I use keyboard navigation to go from the Search bar and eventually to open Outlook Add-in for One Note. From there, you can see the keyboard focus stays on the "..." Onwards, I try to keyboard navigate through the Outlook app to be able to reach inside of the One Note add-in but as you can see, my keyboard focus loops around until I reach the line with all the add-ins again without being able to ever reach inside the add-in.

https://user-images.githubusercontent.com/35610769/152622051-bbdb50ed-f524-405e-90a7-bd435b77827d.mov

**Comment by exextoc** [comment]
Thanks for reporting this issue. 
We are able to reproduce this issue internally. It has been put on our backlog. 
This is the internal tracking id of the bug : 5795803
We, unfortunately, have no timelines to share at this point.

**Comment by exextoc** [comment]
We have fixed the bug and the fix should be available in build 16.0.16204.10000 or later. When an updated build is available depends on the release channel a customer is on.
